frontend/src/lib/components/CommandBar/commandBarLogic.ts

```diff
@@ -19,13 +19,9 @@ export const commandBarLogic = kea<commandBarLogicType>([
                 setCommandBar: (_, { status }) => status,
                 hideCommandBar: () => BarStatus.HIDDEN,
                 toggleSearchBar: (previousState) =>
-                    [BarStatus.HIDDEN, BarStatus.SHOW_SHORTCUTS].includes(previousState)
-                        ? BarStatus.SHOW_SEARCH
-                        : BarStatus.HIDDEN,
+                    previousState === BarStatus.SHOW_SEARCH ? BarStatus.HIDDEN : BarStatus.SHOW_SEARCH,
                 toggleActionsBar: (previousState) =>
-                    [BarStatus.HIDDEN, BarStatus.SHOW_SHORTCUTS].includes(previousState)
-                        ? BarStatus.SHOW_ACTIONS
-                        : BarStatus.HIDDEN,
+                    previousState === BarStatus.SHOW_ACTIONS ? BarStatus.HIDDEN : BarStatus.SHOW_ACTIONS,
```

Comment:
This fixes an issue with toggling between search and actions, see https://posthog.slack.com/archives/C064U4XCPHQ/p1701125167365489.